pkg/util/mount/mount_linux.go

```diff
@@ -323,17 +323,21 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 
 	// Try to mount the disk
 	glog.V(4).Infof("Attempting to mount disk: %s %s %s", fstype, source, target)
-	err = mounter.Interface.Mount(source, target, fstype, options)
-	if err != nil {
+	mountErr := mounter.Interface.Mount(source, target, fstype, options)
```

Comment:
My 2 cents: the problem here is that the function is too long and it should be refactored as opposed to wrong error naming, `err` is fine